packages/jsii/lib/compiler.ts

```diff
@@ -178,7 +178,18 @@ export async function compileSources(entrypoint: string,
 
     if (rootModule) {
         await processModule(rootModule, []);
-        addTypeInfo(assm, types);
+        for (const type of types.sort((l, r) => l.fqn.localeCompare(r.fqn))) {
```

Comment:
Can you not put this in a method? All other calls in this spot are method calls.

Reply:
That's definitely an option. It felt heavy-handed, but I can "restore" `addTypeInfo` and just reduce what it does to just what's needed :)